docs/user-authentication.txt

```diff
@@ -97,8 +97,64 @@ manage user authentication, as shown in the following code:
    :language: php
    :dedent:
 
-Enable Password Reminders 
--------------------------
+Customize User Authentication
+-----------------------------
+
+In this section, you can learn how to use the following features to
+customize your MongoDB user authentication process:
+
+- :ref:`laravel-user-auth-sanctum`
+- :ref:`laravel-user-auth-reminders`
+
+.. _laravel-user-auth-sanctum:
+
+Laravel Sanctum
+~~~~~~~~~~~~~~~
+
+You can install the Laravel Sanctum package to provide API tokens to users
+and authenticate single-page applications. To manage API tokens, Sanctum stores
+each token in a database table and authenticates incoming HTTP requests through
+the ``Authorization`` header. To authenticate single-page applications, Sanctum
+uses Laravel's cookie-based authentication services.
+
+Run the following commands from your project root to install Laravel Sanctum,
+then publish and run its migration files:
+
+.. code-block:: bash
+
+   composer require laravel/sanctum
+   php artisan vendor:publish --provider="Laravel\Sanctum\SanctumServiceProvider"
+   php artisan migrate
```

Comment:
This last command cannot be done before the model is updated to use the `DocumentModel` trait.